pkg/obfuscator/domain.go

```diff
@@ -4,18 +4,21 @@ import (
 	"fmt"
 	"regexp"
 	"strings"
+
+	"github.com/openshift/must-gather-clean/pkg/schema"
 )
 
 const (
-	domainPattern      = `([a-zA-Z0-9\.]*\.)?(%s)`
-	obfuscatedTemplate = "domain%07d"
+	domainPattern                      = `([a-zA-Z0-9\.]*\.)?(%s)`
+	obfuscatedTemplate                 = "domain%07d"
+	staticDomainReplacement            = "obfuscated.com"
+	maximumSupportedObfuscationDomains = 9999999999
```

Comment:
I think you have a couple 9's too many: 9,999,999,999. Your format only allows for seven digits.